### ISSUE TITLE
False-positive F821 on Inner Classes

### ISSUE THREAD

**Issue by dmtucker** [PyCQA/pyflakes#427]
tl;dr: #126 for type annotations

``` python
class A:
    class B:
        pass

    def do(self) -> B:
        return A.B()
```
```
$ python3.7 -m venv tmp
$ tmp/bin/pip install pyflakes
Collecting pyflakes
  Using cached https://files.pythonhosted.org/packages/16/3b/b6a508ad148ce1ef50bd7a9a783afbb8d775616fc4ae5e3007c8815a3c85/pyflakes-2.1.0-py2.py3-none-any.whl
Installing collected packages: pyflakes
Successfully installed pyflakes-2.1.0
You are using pip version 18.1, however version 19.0.1 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
$ tmp/bin/pyflakes tmp.py
tmp.py:5: undefined name 'B'
```
- Python 3.7.2, if it matters

`mypy` has no problem with this annotation:
```
$ tmp/bin/pip install mypy
Collecting mypy
  Downloading https://files.pythonhosted.org/packages/88/11/8092fdd9cf4c507e6c799bf663e713a5418beb9fda422df810f72641224c/mypy-0.670-py3-none-any.whl (1.5MB)
    100% |████████████████████████████████| 1.5MB 5.5MB/s 
Collecting typed-ast<1.4.0,>=1.3.1 (from mypy)
  Downloading https://files.pythonhosted.org/packages/67/da/5ab7377797ecbae3ff08eeffb8183f225bdf4b781ddb832ccf95ebd4f46e/typed_ast-1.3.1-cp37-cp37m-macosx_10_9_x86_64.whl (213kB)
    100% |████████████████████████████████| 215kB 9.0MB/s 
Collecting mypy-extensions<0.5.0,>=0.4.0 (from mypy)
  Using cached https://files.pythonhosted.org/packages/4d/72/8d54e2b296631b9b14961d583e56e90d9d7fba8a240d5ce7f1113cc5e887/mypy_extensions-0.4.1-py2.py3-none-any.whl
Installing collected packages: typed-ast, mypy-extensions, mypy
Successfully installed mypy-0.670 mypy-extensions-0.4.1 typed-ast-1.3.1
You are using pip version 18.1, however version 19.0.1 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
$ tmp/bin/mypy tmp.py 
$ 
```
- Note: I confirmed it would catch it by changing `-> B` to `-> C` and rerunning:
  ```
  $ tmp/bin/mypy tmp.py 
  tmp.py:5: error: Name 'C' is not defined
  ```

**Comment by asottile** [comment]
Can confirm, thanks for the report :+1:

**Comment by Hanaasagi** [comment]
I think this problem was fixed by #448 . You can test with master branch pyflakes.

**Comment by asottile** [comment]
can confirm -- thanks again for the fix @Hanaasagi!
